### ISSUE TITLE
Rename the repo to `mom6-panantarctic` or `panantarctic-mom6`?

### ISSUE THREAD

**Issue by navidcy** [COSIMA/mom6-panan#51]
Why `panan`? It means nothing... it only alienates people.
It consts nothing to have the full word.

**Comment by adele-morrison** [comment]
Hmm, it does match the name of the model simulation output on gadi though. I'm not opposed to changing it though.

**Comment by navidcy** [comment]
You are saying that the `panan` name is now used in many places so it's sorta established?

Because I'd say change it everywhere?

**Comment by AndyHoggANU** [comment]
Yeah, it's tricky to change data filenames ... but actually the data is under `panant`, not `panan`, so clearly mismatches don't matter.

I'd support changing this repo name to `mom6-panantarctic` ...

**Comment by adele-morrison** [comment]
Great, let’s do it then. We have called it PanAntarctic in submitted publications.
